modules/aws-mwaa/main.tf

```diff
@@ -0,0 +1,354 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# MWAA Environment
+# ---------------------------------------------------------------------------------------------------------------------
+resource "aws_mwaa_environment" "mwaa" {
```

Comment:
@drewmullen we should consider creating this as an I&A module?

Reply:
Waiting PR to be reviewed: https://github.com/aws-ia/terraform-aws-mwaa/pull/1
Once it is approved the module will be created under aws-ia org, then I can update the example to point to the new location of the module.